src/actions/breakpoints/addBreakpoint.js

```diff
@@ -174,3 +186,42 @@ export function addBreakpoint(
     });
   };
 }
+/**
+ * Add a new disabled breakpoint
+ *
+ * @memberof actions/breakpoints
+ * @static
+ * @param {String} $1.condition Conditional breakpoint condition value
+ * @param {Boolean} $1.disabled Disable value for breakpoint value
+ */
+
+export function addDisabledBreakpoint(
+  location: SourceLocation,
+  { condition, hidden }: addBreakpointOptions = {}
+) {
+  return async ({ dispatch, getState, sourceMaps, client }: ThunkArgs) => {
+    recordEvent("add_breakpoint");
+
+    if (features.columnBreakpoints && location.column === undefined) {
+      location = getFirstPausePointLocation(getState(), location);
+    }
+
+    const breakpoint = createBreakpoint(location, {
+      condition,
+      hidden,
+      disabled: true
+    });
+
+    return dispatch({
+      type: "ADD_BREAKPOINT",
+      breakpoint,
+      [PROMISE]: addBreakpointPromise(
+        getState,
+        client,
+        sourceMaps,
+        breakpoint,
+        true
```

Comment:
Are there any cases in which `disabled: true` and `shiftKey: false`, or vice versa? 

This seems to be the only place where `disabled: true`, and if that's the case, the `shiftKey` value passed here is redundant as it can just be inferred from `breakpoint.disabled`.

Reply:
add breakpoint is also used with the checkbox in the secondary pane. If someone is enabling the breakpoint by checking the box newBreakpoint needs to be set before assertbreakpoint is called. I didn't add the shiftKey to fix this bug, I added it to fix the bug it created.

Reply:
Oh now I understand, thanks for explaining it. Did not take `toggleBreakpoint` into consideration earlier. In that case this order does make sense 👍